guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -26,6 +26,8 @@ The `{{ site.data.var.ct }}` 2002.0.22 release changes the structure of the `{{
 
    -  {:.new}Updated the `composer.json` file for the `{{ site.data.var.ct }}` package to add a dependency for the `magento/magento-cloud-patches` v1.0.0 package.
 
+   -  {:.fix}<!--MAGECLOUD-4661-->Fixed an issue that caused the `{{ site.data.var.ct }}` patching process to break when applying patch sets that include security-only patches for Magento version 2.3.2 and later . This issue was introduced by the new versioning scheme adopted for [security-only patches]({{ site.baseurl }}/guides/v2.3/release-notes/bk-release-notes.html#security-only-patches) starting with the {{ site.data.var.ee }} version 2.3.2-p2.
```

Comment:
`when applying patch sets that include security-only patches for Magento version 2.3.2 and later ` -> `when applying patch sets on top of security-only releases, starting with Magento 2.3.2-p2 and later`